src/test/java/org/primefaces/PrimeNGSetupPage.yml

```diff
@@ -0,0 +1,11 @@
+urls:
+  default: "https://www.primefaces.org/primeng/setup"
+
+elements:
+  dropdown_button:
+    elementType: Button
+    xpath: "//a[@href='/primeng/dropdown']"
+  get_started_header:
+    xpath: "//h1[text()='Get Started']"
+
+    
```

Comment:
Delete trailing spaces in all files. In yaml files it can cause weird shit to happen but it's just overall unclean.